hypervisor/src/arch/x86/emulator/instructions/mod.rs

```diff
@@ -47,6 +47,89 @@ macro_rules! imm_op {
 
 pub mod mov;
 
+fn get_op<T: CpuStateManager>(
+    insn: &Instruction,
+    n: u32,
+    state: &mut T,
+    platform: &mut dyn PlatformEmulator<CpuState = T>,
+    size: usize,
```

Comment:
Ditto, can we use `op_size` and maybe move it next to `op_index` in the argument list?